articles/paladin_warden/paladin_warden.md

```diff
@@ -0,0 +1,83 @@
+ [Paladin documentation](https://doc.paladin.vote/)  
+ [Paladin Github](https://github.com/PaladinFinance)  
+ [Paladin multisig](https://etherscan.io/address/0x0792dCb7080466e4Bbc678Bdb873FE7D969832B8),
+ [owners](https://gnosis-safe.io/app/eth:0x0792dCb7080466e4Bbc678Bdb873FE7D969832B8/settings/owners)  
+ [Dune Analytics](https://dune.xyz/jjnnbbtt/Paladin.vote)  
+ [Warden Quest Audit](https://github.com/spearbit/portfolio/blob/master/pdfs/Paladin-Spearbit-Security-Review.pdf)  
+ [Paladin Audit](https://github.com/pessimistic-io/audits/blob/914e35fdfc879451a61d2a3969f3be839b8c808f/Paladin%20Security%20Analysis%20by%20Pessimistic.pdf)  
+ [Podcast: Voting Markets, with Paladin's Romain Figuereo](https://anchor.fm/i-pledge-allegiance/episodes/Voting-Markets--with-Paladins-Romain-Figuereo-e1foigm)
+
+## Abstract
+With the introduction of lendable governance power Paladin fills a yet unfilled niche of governance voting markets. In this review we will cover both Paladin and Warden and assess the relevant risks veCRV holders and LPs.
+
+### Paladin protocol overview
+The Paladin protocol allows users to non-custodially lend the governance power of various protocols. The way it operates resembles that of a lending protocol. When a user deposits his tokens into a pool, they are released for borrowing. When a user borrows the governance rights, the [PalPool](https://doc.paladin.vote/paladin-protocol/palpool) creates a [PalLoan](https://doc.paladin.vote/paladin-protocol/palloan) (ERC-721) which represents the borrowing position. The borrowing fees are paid in advance by the user after which the governance rights are delegated to the borrower.
+
+At any time, the borrower can change the recipient of his borrowed governance rights, extend or close the borrowing position. All unused fees will be returned to the borrower should he close his position earlier than initially desired. Conversely, the borrowing position is automatically closed if all fees (that were prepaid and held in custody) are used up.
+
+#### Pools as a faster means of exiting staked gov. power
+Since not every protocol allows users to unstake their governance tokens without a delay, it is possible (e.g. in the case of PalStkAave) to [exit a gov. lending position via a pool](https://gov.paladin.vote/t/using-palstkaave-as-a-liquid-wrapper-for-staked-aave/141). This pool is incentivized by Paladin and serves to improve the user experience of their protocol.
+
+#### Risk summary:
+* **For LPs**
+Significant depegs (e.g. PalStkAave/Aave) do not pose too much risk for liquidity providers, since in the case of a lopsided pool balance the LP can rebalance the pool within 10 days. This can be achieved at no cost by withdrawing PalStkAave, unwrapping, unstaking Aave, and redepositing back into the pool. Ideally, this rebalancing should occur before the pool enters a lopsided state to provide sufficient depth (which translates into volume & fees) at all times.
```

Comment:
I would argue it is both. It is a risk because if you LP PalStkAave/Aave and some PalStkAave user exits through the pool you provided him your (unstaked) Aave as exit liquidity. Say it was whale that exited, the pool is lopsided now. Now if you want to exit your LP and subsequently Aave (sell to USD) you are left with more PalStkAave (which is good) but then you have more capital that needs to go through that 10 days unstaking period.

>>> This "more capital" that needs to through the unstaking period is the smol risk LPs take